src/beautifiers/clojure-beautifier/index.coffee

```diff
@@ -0,0 +1,22 @@
+"use strict"
+path = require('path')
+Beautifier = require('../beautifier')
+
+module.exports = class ClojureBeautifier extends Beautifier
+
+  name: "Clojure Beautifier"
```

Comment:
Thanks for the feedback, I'm happy to change it - I wasn't sure how the name would be used (in the UI etc), so made it generic.